### ISSUE TITLE
关于scaling

### ISSUE THREAD

**Issue by huachao1001** [JUGGHM/OREPA_CVPR2022#3]
作者你好，论文提到用scaling替代bn，但是为什么源码中还是用bn？

**Comment by JUGGHM** [comment]
感谢你的关注！我们在论文中提到用scaling替代dbb block中各个分支上的bn，并且在多分支相加后添加一个额外的post addition bn，这个bn是用来稳定训练的，用于替代的scaling层的作用是使不同分支的优化方向产生差异的。如仍有困惑，欢迎提问。

**Comment by huachao1001** [comment]
作者你好，可能是我没有理解好代码，就是没看到有scaling部分，看到代码里面直接是nn.BatchNorm2d，不知道具体是哪一行代码是作为scaling功能部分？

**Comment by JUGGHM** [comment]
> 作者你好，可能是我没有理解好代码，就是没看到有scaling部分，看到代码里面直接是nn.BatchNorm2d，不知道具体是哪一行代码是作为scaling功能部分？

Scaling层的参数保存在各block类的 `self.vector` 中

**Comment by huachao1001** [comment]
> > 作者你好，可能是我没有理解好代码，就是没看到有scaling部分，看到代码里面直接是nn.BatchNorm2d，不知道具体是哪一行代码是作为scaling功能部分？
> 
> Scaling层的参数保存在各block类的 `self.vector` 中

好的，感谢

**Comment by huachao1001** [comment]
已解决
